docs/data/rpc/api-reference/structure/data-format.mdx

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 60
+title: Data Format
+---
+
+import { ExampleResponse } from "@site/src/components/ExampleResponse";
+import { AttributeTable } from "@site/src/components/AttributeTable";
+
+### XDR Format
+
+In the Stellar network, transactions are encoded using a standardized protocol called [External Data Representation](https://en.wikipedia.org/wiki/External_Data_Representation) (XDR).
+
+In RPC, you will encounter XDR when [simulating](../methods/simulateTransaction) and [sending](../methods/sendTransaction) transactions, as well as when retrieving [transactions](../methods/getTransactions), [ledgers](../methods/getLedgers), and [ledger entries](../methods/getLedgerEntries).
+
+By default, RPC will return all XDR attributes as the machine-readable base64 encoded XDR string. All XDR encoded response fields are suffixed with `Xdr`. You can decode this XDR in the Stellar Lab’s [XDR page](https://lab.stellar.org/xdr/view?$=network$id=testnet&label=Testnet&horizonUrl=https:////horizon-testnet.stellar.org&rpcUrl=https:////soroban-testnet.stellar.org&passphrase=Test%20SDF%20Network%20/;%20September%202015;;).
+
+### JSON Format
+
+All RPC endpoints which return XDR formatted fields accept the `xdrFormat` attribute. This allows a client to change the response format to JSON, ultimately making it more human-readable. Note that you should not rely on any schema for the JSON, as it will change when the underlying XDR changes.
```

Comment:
Nitpick but base64 is the format not XDR heh

```suggestion
All RPC endpoints which return encoded XDR fields accept the `xdrFormat` attribute. This allows a client to change the response format to JSON, ultimately making it more human-readable. Note that you should not rely on any schema for the JSON, as it will change when the underlying XDR changes.
```